aspnetcore/web-api/index.md

```diff
@@ -201,38 +201,6 @@ The `ValidationProblemDetails` type:
 * Provides a machine-readable format for specifying errors in web API responses.
 * Complies with the [RFC 7807 specification](https://tools.ietf.org/html/rfc7807).
 
-To change the default response type to `SerializableError`, apply the highlighted changes in `Startup.ConfigureServices`:
```

Comment:
@scottaddie I thought it would help to move the configuring bits in to one area. What do you think?